app/src/main/res/layout/view_card_on_this_day.xml

```diff
@@ -40,85 +36,94 @@
             android:transitionName="@string/transition_on_this_day"
             tools:text="November 1" />
 
-    </FrameLayout>
+        <View
+            android:id="@+id/gradient_layout"
+            android:layout_width="1dp"
+            android:layout_height="26dp"
+            android:layout_marginStart="27dp"
+            android:layout_marginTop="32dp"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            android:background="?attr/chart_shade5"/>
 
-    <LinearLayout
-        android:id="@+id/view_on_this_day_text_container"
-        android:layout_width="match_parent"
-        android:layout_height="0dp"
-        android:layout_marginBottom="@dimen/margin"
-        android:layout_marginEnd="12dp"
-        android:layout_marginStart="27dp"
-        android:layout_weight="1"
-        android:background="?attr/chart_shade5"
-        android:clipChildren="false"
-        android:clipToPadding="false">
-
-        <RelativeLayout
-            android:id="@+id/view_on_this_day_click_container"
+        <LinearLayout
+            android:id="@+id/view_on_this_day_text_container"
```

Comment:
Feels like the `view_on_this_day_text_container` and `view_on_this_day_click_container` and be combined since the `LinearLayout` contains only one `FrameLayout`

Reply:
The `LinearLayout` is providing the thin dark line on the left (by having a dark background), and then the `FrameLayout` has a light background with a 1dp margin.  If we want to, we can simplify this further in another patch, but this would be too complex for this one.

Reply:
Make senses.